include/mbedtls/x509_crt.h

```diff
@@ -651,4 +651,9 @@ int mbedtls_x509write_crt_pem( mbedtls_x509write_cert *ctx, unsigned char *buf,
 }
 #endif
 
+#ifndef MBEDTLS_TLS_DEFAULT_ALLOW_SHA1_IN_CERTIFICATES
+/* The test infrastructure requires a positive define */
+#define MBEDTLS_X509__DEFAULT_FORBID_SHA1
+#endif
```

Comment:
@andresag01 Correction: someone once tried to do it, but the infrastructure doesn't support it. It wants `!FOO` to be defined, not `FOO` to not be defined. A test with `!FOO` as a requirement is currently never run.